course_content/notebooks/dask_intro.ipynb

```diff
@@ -1241,6 +1241,314 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example lazy operation : Wind calculations\n",
+    "\n",
+    "#### First grab some sample wind vector data\n",
+    "This is in cubes of U and V (i.e. eastward and northward winds)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filepath = iris.sample_data_path('wind_speed_lake_victoria.pp')\n",
+    "u_cube, v_cube = iris.load_cubes(filepath, ('x_wind', 'y_wind'))\n",
+    "\n",
+    "print 'U cube : \\n', u_cube\n",
+    "print\n",
+    "print 'V cube : \\n', v_cube.summary(shorten=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### These cubes have lazy data initially, which means they actually contain Dask arrays."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print 'U cube : lazy =', u_cube.has_lazy_data()\n",
+    "print 'U cube : raw data content =', u_cube.core_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Iris in-built lazy operations : statistics\n",
+    "\n",
+    "Let's take average winds over the region.  \n",
+    "This uses the Iris Cube.collapsed() operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mean_u = u_cube.collapsed(('latitude','longitude'), iris.analysis.MEAN)\n",
+    "mean_v = v_cube.collapsed(('latitude','longitude'), iris.analysis.MEAN)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As the input cubes have lazy data, and the MEAN operator is lazy-preserving, the results' data should still be deferred calculations -- i.e. the source data has not yet been loaded from disk.\n",
+    "\n",
+    "Check this out, as an exercise ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exercise: check lazy status\n",
+    "\n",
+    "**1.** Show that the above results have not yet been calculated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**2.** Print the mean wind values "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**3.**  \n",
+    "Have the data been loaded ?  \n",
+    "Are the mean cubes still lazy ?  \n",
+    "What about the original u/v cubes ?  "
```

Comment:
This question is relative and not explicit. It would be better to ask:
"Are the original u/v cubes still lazy?"

For the sake of flow it could be partially relative without losing its explicit quality, such as:
"Are the original u/v cubes also still lazy?"